### DISCUSSION TITLE
Updated Panel Uplad process

### DISCUSSION THREAD

**Discussion by JamesHagerman** [RomanKubiak/ctrlr#363] [Ideas]
Hello!

I just started using Ctrlr to control my Ensoniq ESQ-1 and have been improving/extending the previously uploaded panel to include the ability to load patches from the synth (the existing version only sends data to the synth).

Now that I have started making good progress, I would love to share my work with the broader community.

If I create a PR which places my latest panel in a new subdirectory, is there any chance of it being accepted into the repo?

This style of contribution is well underway in other open source projects (like in the open source mechanical keyboard firmware project, QMK).

If not, what is the new, GitHub-friendly way of uploading panels?

Thank you for the help,
James

**Comment by tomlem** [comment]
I had to think about it for a while. 
I suggest that new panels be uploaded in Github as a zip file under **>Discussions>Releases**.

**Reply by damiensellier** [reply]
That's a good idea +1

**Reply by bijlevel** [reply]
Roman  seems to have made in the past a place to upload panels on Github: https://github.com/RomanKubiak/Panels, but there is not much sign of life over there : ;-)! So indeed better to do it on this repository. Or else on the old ctrlr forum? It seems that people still visit the download page...

**Reply by Godlike-Productions** [reply]
I'm pretty sure https://github.com/RomanKubiak/Panels, is linked from this repo, so that would be the best place to put them

**Reply by Godlike-Productions** [reply]
There's also preset patch, which has a section for Ctrlr panels
https://www.presetpatch.com/articles/CTRLR-Editors,-VSTs-and-Panel-Saving-Tools
